src/components/periodic-report/periodic-report.repository.ts

```diff
@@ -39,6 +39,8 @@ export class PeriodicReportRepository extends DtoRepository(IPeriodicReport) {
       type: input.type,
       start: input.start,
       end: input.end,
+      skipped: input.skippedReason != null,
```

Comment:
Err sorry I mean that in general. We don't have to write this to the DB as we can check it on read.